### PR TITLE
Progres of details component, added enums for ticket status and even…

### PR DESCRIPTION
**Pull request by mileTadic1994**
WIP - This pull request is still in progress.

I've finished design of details view for events.

Still need to do:

1. Change icons when @simple-taskic or anyone else provide me from sketch, since I don't have that tool.
2. There are three different cases for rendering button/text label depending on Event Status(Sold out, Available and Not on Sale Yet). What are the other cases?

3. Still need to implement Ticket button action.

@Krakaw Please, can you provide me answers till tomorrow? Also I would like you to review design and provide me additional request if you have them.

### CODE REVIEW

**Comment by simple-taskic** [comment]
Hey @mileTadic1994 please can you create a proper issue with some description and refer in the description of this PR so that we are aware of the scope?

File: src/Events/details.js [hunk 1]
```diff
@@ -1,246 +1,111 @@
...
+    const { event } = this.props;
+    const { status, ticket_pricing, sales_start_date } = event
+    if (!sales_start_date) return null
+    if (status === TICKET_STATUS.PUBLISHED) {
+      return (
+        <View style={[styles.button, styles.margin]}>
+          <LinearGradient
+            start={{ x: 0, y: 0 }}
+            end={{ x: 1, y: 0 }}
+            colors={['#5491CC', '#9A68B2', '#E53D96']}
```
Comment (simple-taskic) [inline src/Events/details.js]: Can we put these colors in some constants with proper names?

File: src/Events/show.js [hunk 2]
```diff
@@ -416,23 +297,26 @@ export default class EventShow extends Component {
...
           onDidFocus={() => this.loadEvent()}
           onWillBlur={() => this.clearEvent()}
         />
         <LoadingScreen visible={showLoadingModal} />
         <SuccessScreen visible={showSuccessModal} />
         <Image
-          style={eventDetailsStyles.videoBkgd}
+          style={isDetailsScreen ? eventDetailsStyles.coverBackground : eventDetailsStyles.videoBkgd}
           source={{uri: optimizeCloudinaryImage(event.promo_image_url)}}
+          blurRadius={isDetailsScreen ? 3 : 0}
```
Comment (simple-taskic) [inline src/Events/show.js]: Maybe we can put this number in a proper constant.

File: src/Events/details.js [hunk 3]
```diff
@@ -1,246 +1,111 @@
...
+    const { ticket_types } = event;
+    if (ticket_types) {
+      return (
+        <View style={eventDetailsStyles.priceContiner}>
+          <Text style={eventDetailsStyles.cardItemTitle}>TICKETS</Text>
+          {
+            ticket_types.map(({ id, name, ticket_pricing }) => (
+              <View key={id} style={eventDetailsStyles.priceItem}>
+                <Text style={event.override_status !== TICKET_STATUS.SOLD_OUT ? eventDetailsStyles.price : eventDetailsStyles.priceDisabled}>
+                  {`$${toDollars(ticket_pricing.price_in_cents - ticket_pricing.discount_in_cents, 0)}`}
```
Comment (simple-taskic) [inline src/Events/details.js]: Can we subtract `ticket_pricing.price_in_cents - ticket_pricing.discount_in_cents` and put result value in some constant?

**Comment by mileTadic1994** [comment]
> Hey, @mileTadic1994 please can you create a proper issue with some description and refer to this PR so that we can be aware what is the scope?

File: src/Events/show.js (hunk 2 above)
Comment (mileTadic1994) [reply src/Events/show.js]: I dont think we should use constants for this, since its just value of blurring. Its used just here

**Pull request closed without merging**